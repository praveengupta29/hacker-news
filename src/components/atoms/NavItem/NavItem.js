import React from 'react';
import PropTypes from 'prop-types';

import './NavItem.scss';

const NavItem = ({ children }) => {
  return <div>{children}</div>;
};

NavItem.propTypes = {
  children: PropTypes.node.isRequired,
};

export default NavItem;
