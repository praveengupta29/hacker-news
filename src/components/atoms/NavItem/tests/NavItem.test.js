import React from 'react';
import { shallow } from 'enzyme';
import NavItem from '../NavItem';

describe('NavItem component', () => {
  const props = {
    children: 'new',
  };

  it('should render correctly and match snapshot', () => {
    const component = shallow(<NavItem {...props} />);
    expect(component).toMatchSnapshot();
  });
});
