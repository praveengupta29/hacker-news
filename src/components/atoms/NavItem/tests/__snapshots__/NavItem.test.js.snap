// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItem component should render correctly and match snapshot 1`] = `
<div>
  new
</div>
`;
